iOS/LocalPackages/DataBrokerProtection-iOS/Sources/DataBrokerProtection-iOS/ManagingAndCoordinating/DataBrokerProtectionIOSManager.swift

```diff
@@ -416,8 +414,13 @@ extension DataBrokerProtectionIOSManager: DBPIOSInterface.RunPrerequisitesDelega
 
     public func validateRunPrerequisites() async -> Bool {
         do {
-            if !(try meetsProfileRunPrequisite) || !meetsAuthenticationRunPrequisite {
-                Logger.dataBrokerProtection.log("Prerequisites are invalid")
+            guard try meetsProfileRunPrequisite else {
```

Comment:
Things got a bit ugly when updating the original logic to use an `await` so I just broke these into two standalone checks, though I admit this is just personal preference on my part, happy to change it.